backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/PrototypeCleanupServiceTest.java

```diff
@@ -0,0 +1,178 @@
+package de.bund.digitalservice.ris.norms.application.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import de.bund.digitalservice.ris.norms.domain.entity.Fixtures;
+import de.bund.digitalservice.ris.norms.domain.entity.Metadata;
+import de.bund.digitalservice.ris.norms.domain.entity.Norm;
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+class PrototypeCleanupServiceTest {
+
+  private final PrototypeCleanupService underTest = new PrototypeCleanupService();
```

Comment:
nitpick: in the other tests we have always named this by the name of the service (so eg. `prototypeCleanupService`)